README.md

```diff
@@ -14,59 +20,50 @@ Please let us know if you find this repository useful. Your feedback will help u
 
 ## Install
 
+Ipyannotator is available on Pypi and can be installed using:
+
 `pip install ipyannotator`
 
-**dependencies (should be handled by pip)**
+## Running ipyannotator
 
-```
-python = "^3.7"
-traitlets = '=4.3.3'
-ipycanvas = "^0.5.1"
-ipyevents = "^0.8.0"
-ipywidgets = "^7.5.1"
-```
+Ipyannotator provides a [simple API](https://palaimon.github.io/ipyannotator/nbs/21_api_doc.html) that provides the ability to explore, create and improve annotation datasets by using a pair of input/outputs. All pair of input/output are [listed on Ipyannotator's docs](https://palaimon.github.io/ipyannotator/nbs/22_input_output_doc.html). Check [Ipyannotator tutorials](https://palaimon.github.io/ipyannotator/docs/tutorials.html) for a quickly demonstration of the library.
 
-## Run ipyannotator as stand-alone web app using voila
+### Run ipyannotator tests
 
-Using `poetry`:
+The notebooks contains the development dependencies: [pytest](https://docs.pytest.org/en/7.1.x/) and [ipytest](https://github.com/chmp/ipytest). The tests can be executed using `nbdev_test_nbs` on the terminal. 
+
+### Run ipyannotator as stand-alone web app using voila
+
+Ipyannotator can be executed as a web app using the [voila](https://github.com/voila-dashboards/voila) library. The following sections describe how to run using poetry and pip.
```

Comment:
> describe how to run using poetry and pip.

describe how to run the voila app using poetry or pip.

Reply:
@ibayer the following sections describe how to run using poetry or pip.

Let me know if your previous suggestions were also solved.